src/store.coffee

```diff
@@ -16,7 +16,8 @@ module.exports = (session) ->
 		###
 		
 		constructor: (config, options) ->
-			@table = options.table if options?.table
+			@table = "[" + options.table + "]" if options?.table
```

Comment:
Ok. So you'd leave it up to the user to use square brackets if the schema or table has spaces / special characters.